azure_sdk/lib/azure_sdk/version.rb

```diff
@@ -0,0 +1,7 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+module Azure
+    VERSION = '0.5.0' # File.read(File.expand_path('../../../../ARM_VERSION', __FILE__)).strip
```

Comment:
Yup not required.

---
In reply to: [76344267](https://github.com/Azure/azure-sdk-for-ruby/pull/433#discussion_r76344267) [](ancestors = 76344267)